### PR TITLE
include org plan in the log message

### PR DESCRIPTION
**Pull request by ruthlesshelp**
Contributes to lanetix/issues#12569

Depends on https://github.com/lanetix/auth/pull/174

Writes log messages that include the plan, such as, `req=45040648-3efb-4cfa-bdec-72f8a905207b, user=2980508, org=2760809, plan: DEV`

With this ☝️ format, PaperTrail Alerts can be edited to _not send_ alerts to PagerDuty if by adding exclusions, such as:
`-"plan= DEV" -"plan= DEMO"`

### CODE REVIEW

File: middleware/log.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ module.exports = function (req, res, next) {
 
     if (user) {
       response.push('org: ' + user.organization_id);
+      response.push('plan: ' + user.organization_plan);
```
Comment (kyspy) [inline middleware/log.js]: Are we guaranteed to always have a plan on the user object?

Comment (tone81) [reply middleware/log.js]: `plan` can be `null` and has no default value from what I'm seeing.

Comment (tone81) [reply middleware/log.js]:
```sql
-- auto-generated definition
CREATE TABLE insecure_organizations
(
  id              INTEGER DEFAULT nextval('snowflake_seq' :: REGCLASS)        NOT NULL
    CONSTRAINT organizations_pkey
    PRIMARY KEY,
  name            TEXT                                                        NOT NULL,
  inbox_key       TEXT                                                        NOT NULL
    CONSTRAINT organizations_inboxkey_unique
    UNIQUE,
  system_user_id  INTEGER
    CONSTRAINT organizations_system_user_id_foreign
    REFERENCES users
    ON UPDATE CASCADE ON DELETE SET NULL,
  permission_sets TEXT [] DEFAULT '{}' :: TEXT []                             NOT NULL,
  settings        JSONB DEFAULT '{}' :: JSONB                                 NOT NULL,
  status          ORGANIZATION_STATUS DEFAULT 'active' :: ORGANIZATION_STATUS NOT NULL,
  created_at      TIMESTAMP WITH TIME ZONE DEFAULT now()                      NOT NULL,
  plan            TEXT,
  mrr             NUMERIC,
  labels          TEXT [] DEFAULT '{}' :: TEXT []                             NOT NULL,
  renews_at       TIMESTAMP WITH TIME ZONE,
  timezone        TEXT DEFAULT 'UTC' :: TEXT                                  NOT NULL
    CONSTRAINT insecure_organizations_timezone_check
    CHECK ('' :: TEXT <> (timezone(timezone, now())) :: TEXT)
);
```

Comment (ruthlesshelp) [reply middleware/log.js]: This is dependent on this https://github.com/lanetix/auth/pull/174/files
which I believe puts the `organization_plan` on the user object.
Is my understanding correct? Am I missing something?

**Comment by ruthlesshelp** [comment]
@tone81 - I was thinking we would deploy this to all our microservices.

File: middleware/log.js (hunk 1 above)
Comment (tone81) [reply middleware/log.js]: Missing nothing. At worst case, we're going to see `null` there, which won't harm anything.

**Comment by tone81** [comment]
Got it. Consider using https://github.com/lanetix/lxstartplus#lxrevdep or going through [this file](https://github.com/lanetix/lxstartplus/blob/master/microservice.json) when targeting where we need to upgrade.

**Pull request merged**